apiary.apib

```diff
@@ -985,6 +985,192 @@ Outputs the list of peers that were removed.
 
 # Group get
 
+## get [POST /get{?arg}{&output,archive,compress,compression%2Dlevel}]
+Download IPFS objects.
+
+Retrieves the object named by <ipfs-or-ipns-path> and stores the data to disk.
+
++ Parameters
+    + arg (string, required) - The path to the IPFS object or objects to be outputted
+    + output (string, optional) - The path where output should be stored. Default: ./<ipfs-path>.
```

Comment:
- where **the** output should be stored.
- Shouldn't the default value be in backticks?